docs/core/project-sdk/msbuild-props.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: MSBuild properties for Microsoft.NET.Sdk
```

Comment:
@gewarren, I think this have a titleSuffix ".NET Core".

I'll open a follow-up PR